packages/core/src/application_ref.ts

```diff
@@ -266,6 +266,24 @@ export interface BootstrapOptions {
    * the change detection will only be trigged once.
    */
   ngZoneEventCoalescing?: boolean;
+
+  /**
+   * Optionally specify coalescing change detections or not.
+   * Consider the following case.
+   *
+   * for (let i = 0; i < 10; i ++) {
+   *   ngZone.run(() => {
+   *     // do something
+   *   });
+   * }
+   *
+   * This case triggers the change detection multiple times.
+   * So with ngZoneCoalescing options, all change detections in one
+   * event loop will only be triggered once, also the change detection
+   * will be executed in requestAnimation just like `markDirty()`.
```

Comment:
@kapunahelewong can you help with making this documentation more consistent with angular.

Reply:
How about this, @JiaLiPassion? 

```suggestion
This case triggers the change detection multiple times.
With ngZoneCoalescing options, all change detections in an event loops trigger only once.
In addition, the change detection executes in requestAnimation, just like in markDirty().
```

These are @aikidave's words. Thanks, Dave!

Reply:
Got it, thank you!